.ci/setup_env.sh

```diff
@@ -20,7 +20,15 @@ BUILD_TOOLS_DOWNLOAD=$DOWNLOAD_ROOT/kong-build-tools
 BUILD_TOOLS_BRANCH=${BUILD_TOOLS_BRANCH:=master}
 
 if [ ! -d $BUILD_TOOLS_DOWNLOAD ]; then
-    git clone -b $BUILD_TOOLS_BRANCH https://github.com/Kong/kong-build-tools.git $BUILD_TOOLS_DOWNLOAD
+    #git clone -b $BUILD_TOOLS_BRANCH https://github.com/Kong/kong-build-tools.git $BUILD_TOOLS_DOWNLOAD
+
+    # temporary operation, get pr430
```

Comment:
Probably this means pr440 now? https://github.com/Kong/kong-build-tools/pull/440